### ISSUE TITLE
Never-winners and choice of min_gap/max_gap

### ISSUE THREAD

**Issue by tmeling** [setzler/eventStudy#11]
When I set  min_control_gap = 1 and max_control_gap = Inf, the package allows me to include never-winners in the sample by using never_treat_action = "keep". 

However, if I set min_control_gap = 1 and max_control_gap =1, the code breaks down. Below, I have copy-pasted my code and the resulting error message.

results <- ES( long_data = event_study, outcomevar = "logsales",  unit_var = "tin", cluster_vars="tin", cal_time_var = "tax_yr", onset_time_var = "win_yr", omitted_event_time = -2, min_control_gap = 1, max_control_gap = 1 , control_subset_var=NA, control_subset_event_time=0, never_treat_action = "keep")

INFO [2019-02-15 09:14:52] Beginning ES.
INFO [2019-02-15 09:14:52] Beginning data stacking.
Error in `[.data.table`(balanced_treated_control, , na.omit(c(outcomevar,  : 
  column(s) not found: logsales, tin, tax_yr, win_yr, ref_onset_time, ref_event_time, catt_specific_sample, treated
In addition: Warning message:
In max(get(cal_time_var)) : no non-missing arguments to max; returning -Inf

**Comment by setzler** [comment]
@tmeling What is the desired behavior? Do you want the control group to be the never-treated group and those who are treated at event time +1 only? 

Our thinking has been that there are three possible cases:
- only use the never-treated control group (never_treat_action = "only", max_control_gap is ignored),
- only use those with finite treatment year as the control group (max_control_gap < Inf and never_treat_action = "exclude"),
- use both never-treated and no upper bound on treatment year (max_control_gap = Inf and never_treat_action = "keep").

**Comment by tmeling** [comment]
Yes, that is correct. I want the control group to include both never-winners and those that eventually win, and I want to loop across various specifications of min_control_gap and max_control_gap to check whether the results differ across specifications.

**Comment by davidnov** [comment]
@tmeling I think this should be possible, but can you explain a bit more a setting where you would want to use, say, those that win shortly after the treatment group as well as those that never win (but exclude those that win long after the treatment group)?

I understand in your exercise (kind of a robustness check), you are just checking something like all combinations of not-yet winners and never winners, but would be useful to have a concrete example where you would really be interested in some of those combinations of control groups.
